### DISCUSSION TITLE
Mappings not saving on blueretro.io - N64 Adapter

### DISCUSSION THREAD

**Discussion by nwad2** [darthcloud/BlueRetro#858] [Q&A]
Running current firmware 1.9.1, but mappings for the N64 Adapter don't seem to save when I change them on blueretro.io. After hitting save, I get the message that the mapping was saved, but if I reconnect the adapter, it has been reverted to the defaults. Are there any known mapping issues with the N64 Adapter?

**Comment by darthcloud** [comment]
Work fine for me.

Maybe try to delete the config file via the file manager:
https://blueretro.io/files.html

Then reboot adapter, give it some time to recreate the file like 30sec to be sure.

**Comment by nwad2** [comment]
I figured it out. I was misinterpreting the labels reverting to the mappings reverting. I wasn't looking closely enough.
